public/data/ChoudharyTara.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "Tara Choudhary",
+  "bio": "I'm Passionate and enthusiast about exploring new things and web development",
```

Should be lowercase, (Punctuations in English)

```suggestion
  "bio": "I'm passionate and enthusiast about exploring new things and web development",
```

---

Done!